src/includes/class-wordlift.php

```diff
@@ -389,10 +389,14 @@ private function define_admin_hooks() {
 		$this->loader->add_action( 'save_post', $this->entity_service, 'save_post', 9, 3 );
 		$this->loader->add_action( 'edit_form_before_permalink', $this->entity_service, 'edit_form_before_permalink', 10, 1 );
 
-		// Add custom columns for entity listing in the backand
+		// Entity listing customization (wp-admin/edit.php)
+		// Add custom columns
 		$this->loader->add_filter( 'manage_entity_posts_columns', $this->entity_list_service, 'register_custom_columns' );
 		$this->loader->add_filter( 'manage_entity_posts_custom_column', $this->entity_list_service, 'render_custom_columns', 10, 2 );
-
+		// Add 4W selection
+		$this->loader->add_action( 'restrict_manage_posts', $this->entity_list_service, 'add_4W_filter' );
+		$this->loader->add_filter( 'posts_clauses', $this->entity_list_service, 'add_4W_filter_query' );
```

Comment:
I would name the methods after the actions/filters names to make it easy to understand the binding between the method and the action/filter.